tools/update-components.sh

```diff
@@ -13,107 +13,107 @@ TFLITE_REPO_URL="https://github.com/espressif/tflite-micro-esp-examples.git"
 
 #this is a temp measure to fix build issue
-if [ -f "$AR_COMPS/esp-rainmaker/components/esp-insights/components/esp_insights/scripts/get_projbuild_gitconfig.py" ] && [ `cat "$AR_COMPS/esp-rainmaker/components/esp-insights/components/esp_insights/scripts/get_projbuild_gitconfig.py" | grep esp32c6 | wc -l` == "0" ]; then
-	echo "Overwriting 'get_projbuild_gitconfig.py'"
-	cp -f "tools/get_projbuild_gitconfig.py" "$AR_COMPS/esp-rainmaker/components/esp-insights/components/esp_insights/scripts/get_projbuild_gitconfig.py"
-fi
+#if [ -f "$AR_COMPS/esp-rainmaker/components/esp-insights/components/esp_insights/scripts/get_projbuild_gitconfig.py" ] && [ `cat "$AR_COMPS/esp-rainmaker/components/esp-insights/components/esp_insights/scripts/get_projbuild_gitconfig.py" | grep esp32c6 | wc -l` == "0" ]; then
+#       echo "Overwriting 'get_projbuild_gitconfig.py'"
+#       cp -f "tools/get_projbuild_gitconfig.py" "$AR_COMPS/esp-rainmaker/components/esp-insights/components/esp_insights/scripts/get_projbuild_gitconfig.py"
+#fi
 
 #
 # CLONE/UPDATE ESP-LITTLEFS
 #
-echo "Updating ESP-LITTLEFS..."
-if [ ! -d "$AR_COMPS/esp_littlefs" ]; then
-	git clone $LITTLEFS_REPO_URL "$AR_COMPS/esp_littlefs" && \
-    git -C "$AR_COMPS/esp_littlefs" submodule update --init --recursive
-else
-	git -C "$AR_COMPS/esp_littlefs" fetch && \
-	git -C "$AR_COMPS/esp_littlefs" pull --ff-only && \
-    git -C "$AR_COMPS/esp_littlefs" submodule update --init --recursive
-fi
-if [ $? -ne 0 ]; then exit 1; fi
+#echo "Updating ESP-LITTLEFS..."
+#if [ ! -d "$AR_COMPS/esp_littlefs" ]; then
+#       git clone $LITTLEFS_REPO_URL "$AR_COMPS/esp_littlefs" && \
+#    git -C "$AR_COMPS/esp_littlefs" submodule update --init --recursive
+#else
+#       git -C "$AR_COMPS/esp_littlefs" fetch && \
+#       git -C "$AR_COMPS/esp_littlefs" pull --ff-only && \
+#    git -C "$AR_COMPS/esp_littlefs" submodule update --init --recursive
+#fi
+#if [ $? -ne 0 ]; then exit 1; fi
 
 #
 # CLONE/UPDATE TINYUSB
 #
 echo "Updating TinyUSB..."
 if [ ! -d "$AR_COMPS/arduino_tinyusb/tinyusb" ]; then
-	git clone $TINYUSB_REPO_URL "$AR_COMPS/arduino_tinyusb/tinyusb"
+        git clone $TINYUSB_REPO_URL "$AR_COMPS/arduino_tinyusb/tinyusb"
 else
-	git -C "$AR_COMPS/arduino_tinyusb/tinyusb" fetch && \
-	git -C "$AR_COMPS/arduino_tinyusb/tinyusb" pull --ff-only
+        git -C "$AR_COMPS/arduino_tinyusb/tinyusb" fetch && \
+        git -C "$AR_COMPS/arduino_tinyusb/tinyusb" pull --ff-only
 fi
 if [ $? -ne 0 ]; then exit 1; fi
 
 #
 # CLONE/UPDATE TFLITE MICRO
 #
-echo "Updating TFLite Micro..."
-if [ ! -d "$AR_COMPS/tflite-micro" ]; then
-	git clone $TFLITE_REPO_URL "$AR_COMPS/tflite-micro"
-	git -C "$AR_COMPS/tflite-micro" submodule update --init --recursive
-else
-	git -C "$AR_COMPS/tflite-micro" fetch && \
-	git -C "$AR_COMPS/tflite-micro" pull --ff-only
-	git -C "$AR_COMPS/tflite-micro" submodule update --init --recursive
-fi
-if [ $? -ne 0 ]; then exit 1; fi
+#echo "Updating TFLite Micro..."
```

Comment:
see my comment about esp32-camera. We do not need those to be required by the component?

Reply:
It is already in the yml file. The libraries are built as espressif__tftlite-micro.a